src/schedule/schedule_inner.rs

```diff
@@ -148,3 +152,19 @@ impl Schedule {
 		}
 	}
 }
+
+/// Get schedule JSON from definition file
+pub fn get_schedule_from_config() -> ScheduleDefinition {
+	// Load the definition.
+	let string = if cfg!(target_arch = "wasm32") {
+		include_str!("../../def.json").to_string()
```

Comment:
This will fail to compile because you removed the default schedule file.

Reply:
Fixed